test/unit/methods.spec.ts

```diff
@@ -740,6 +741,28 @@ describe("Methods call", () => {
         50,
       );
     });
+
+    it("shouldn't move camera position when it's freeScroll mode", async () => {
+      // Given
+      flickingInfo = createFlicking(horizontal.full, {
+        moveType: "freeScroll",
+      });
+      const flicking = flickingInfo.instance;
+      const viewport = (flicking as any).viewport as Viewport;
+      await simulate(flickingInfo.element, {
+        deltaX: -200,
+        duration: 100,
+      });
+      await waitEvent(flicking, "moveEnd");
+      const prevCameraPosition = viewport.getCameraPosition();
+
+      // When
+      flicking.resize();
+
+      // Then
+      const afterCameraPosition = viewport.getCameraPosition();
+      expect(prevCameraPosition).equals(afterCameraPosition);
+    });
   });
```

Comment:
@WoodNeck 
Wouldn't it be nice to have a test when the pos goes out of range?

Reply:
That can be temporary fix.
To fix it correctly, we should modify Axes first so we can animating position as our own.
I think we should be able to change camera position while resizing first.